autoload/lsp/diag.vim

```diff
@@ -362,7 +362,7 @@ export def LspDiagsJump(lspserver: dict<any>, which: string): void
     if (which == 'next' && (lnum > curlnum || lnum == curlnum && col > curcol))
 	  || (which == 'prev' && (lnum < curlnum || lnum == curlnum
 							&& col < curcol))
-	  || (which == 'here' && (lnum == curlnum && col >= curcol))
+	  || (which == 'here' && (lnum == curlnum && col > curcol))
```

Comment:
With this change, if there is a diagnostic at the beginning of a line and the cursor is at the first column, then the ":LspDiagHere" command will fail and there is no way to jump to this diagnostic.  If there are multiple diagnostics
on a single line, you can use the ":LspDiagNext" command after the ":LspDiagHere" command to jump to the
next diagnostic on the line.  With this change, this will command will behave like ":LspDiagAfter".